aptos-move/move-examples/post_mint_reveal_nft/sources/minting.move

```diff
@@ -85,8 +106,14 @@ module post_mint_reveal_nft::minting {
         property_types: vector<String>,
     }
 
-    /// WhitelistMintConfig stores information about whitelist minting.
+    /// WhitelistMintConfig stores information about all stages of whitelist.
+    /// Most whitelists are one-stage, but we allow multiple stages to be added in case there are multiple rounds of whitelists.
     struct WhitelistMintConfig has key {
```

Comment:
I feel like it'd be a lot easier to follow if we separate everything whitelist into a separate module.

Reply:
yes that's true, but we also don't know in advance how many stages there are?

Reply:
good point, will make it a separate module